pkg/flag/misconf_flags.go

```diff
@@ -43,6 +43,12 @@ var (
 		Value:      []string{},
 		Usage:      "specify paths to override the Terraform tfvars files",
 	}
+	K8sVersionFlag = Flag{
```

Comment:
I understand this flag is for misconfiguration detection, but we already have the k8s flag group. What if moving this flag there?